src/content/notInNavigation/accessibility.md

```diff
@@ -89,3 +89,32 @@ Chromatic builds upon Storybook's accessibility testing by providing automated t
 No, automated testing catches a subset of accessibility issues. Manual testing is still recommended for comprehensive accessibility compliance.
 
 </details>
+
+<details>
+<summary>Why don't the accessibility results show violations, even though I know there are some?</summary>
+
+Modern React components often use asynchronous techniques like Suspense or React Server Components (RSC) to handle complex data fetching and rendering. These components don’t immediately render their final UI state. Storybook doesn’t inherently know when an async component has fully rendered. As a result, the a11y addon sometimes run too early, before the component finishes rendering, leading to false negatives (no reported violations even if they exist).
+
+This only impacts users using:
+* Storybook 8.5+
+* Async components (RSC, or using Suspense or similar)
+* Stories without a play function that awaits for the final component state
+
+To address this issue, enable the `developmentModeForBuild` feature flag. This sets `process.env.NODE_ENV` to `development` when building your Storybooks. This ensures that React’s `act` utility is used, which helps ensure that all updates related to a test are processed and applied before making assertions.
```

Comment:
FYI, I suggested some improvements for the SB docs which apply here as well:

https://github.com/storybookjs/storybook/pull/30037/files#r1894080439